api/src/main/java/io/opencensus/trace/export/ExportComponent.java

```diff
@@ -45,19 +45,19 @@ public static ExportComponent getNoopExportComponent() {
   public abstract SpanExporter getSpanExporter();
 
   /**
-   * Returns the {@link ActiveSpans} that can be used to get useful debugging information about all
-   * the current active spans.
+   * Returns the {@link ActiveSpansExporter} that can be used to get useful debugging information
+   * about all the current active spans.
    *
-   * @return the {@code ActiveSpans} or {@code null} if not supported.
+   * @return the {@code ActiveSpansExporter} or {@code null} if not supported.
    */
   @Nullable
-  public abstract ActiveSpans getActiveSpans();
+  public abstract ActiveSpansExporter getActiveSpansExporter();
```

Comment:
I just noticed that the pluralization of "span" is inconsistent; the classes are SpanExporter, SampledSpanStore, and ActiveSpansExporter.  I don't know if that's a problem, and I don't think it needs to be changed in this PR.